thruster_interface/scripts/thruster_interface_node.py

```diff
@@ -175,10 +190,10 @@ def validate_and_limit_thrust(self, thrust_msg):
         return Float32MultiArray(data=thruster_forces)
 
     def limit_pwm(self, pwm):
-        if pwm > 1900:
-            return 1900
-        if pwm < 1100:
-            return 1100
+        if pwm > self.pwm_limit_max:
+            return self.pwm_limit_max
+        if pwm < self.pwm_limit_min:
+            return self.pwm_limit_min
```

Comment:
I'd like the thruster limiting to be re-implemented as a percentage-based value instead.